### PR TITLE
PLATUI-1265: Update examples for govuk-frontend v3.13.0

### PR DESCRIPTION
**Pull request by matthewmascord**

### CODE REVIEW

File: src/it/scala/uk/gov/hmrc/hmrcfrontend/views/example/hmrcAddToAListIntegrationSpec.scala [hunk 1]
```diff
@@ -14,9 +14,7 @@ class hmrcAddToAListIntegrationSpec extends TemplateIntegrationSpec {
   val request: RequestHeader = FakeRequest()
   val messages: Messages     = MessagesSupport().messages
```
Comment (JoPintoPaul) [inline src/it/scala/uk/gov/hmrc/hmrcfrontend/views/example/hmrcAddToAListIntegrationSpec.scala:15]: Can these values `request` and `messages` be deleted as those tests are no longer run?

File: src/test/play-2/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/addToSummary.scala.html [hunk 2]
```diff
@@ -1,38 +0,0 @@
-@import uk.gov.hmrc.hmrcfrontend.views.html.components._
```
Comment (JoPintoPaul) [inline src/test/play-2/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/addToSummary.scala.html:1]: @matthewmascord Has this example been deleted?

File: src/it/scala/uk/gov/hmrc/hmrcfrontend/views/example/hmrcAddToAListIntegrationSpec.scala (hunk 1 above)
Comment (matthewmascord) [reply src/it/scala/uk/gov/hmrc/hmrcfrontend/views/example/hmrcAddToAListIntegrationSpec.scala]: Re-instated applicable tests.

File: src/test/play-2/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/addToSummary.scala.html (hunk 2 above)
Comment (matthewmascord) [reply src/test/play-2/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/addToSummary.scala.html]: Re-instated deleted examples.

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**